src/server/db/database.js

```diff
@@ -0,0 +1,45 @@
+import 'dotenv/config';
+import pgp from 'pg-promise';
+
+class Database {
+  constructor() {
+    const connection = {
+      host: 'v9_dep_mgr.patcody.io',
+      port: 5432,
+      database: 'v9',
+      user: process.env.POSTGRES_USERNAME,
+      password: process.env.POSTGRES_PASSWORD
+    };
+    this.db = pgp()(connection);
```

Comment:
Error handling?

Reply:
It's lazy-loaded, so I'm reasonably sure this should just work- I can surround with try-catch block if needed

Reply:
If it's lazy it's good as is